BUILD.bazel

```diff
@@ -8,7 +8,7 @@
 
 load("@rules_proto//proto:defs.bzl", "proto_library")
 load("@rules_python//python:defs.bzl", "py_library", "py_runtime", "py_runtime_pair")
-load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library", "cc_proto_library", "cc_test")
+load("@rules_cc//cc:defs.bzl", "cc_library", "cc_binary", "cc_proto_library", "cc_test")
```

Comment:
nit: why the order of loading needs to change? seems that it was alphabetically ordered before?

Reply:
Still iterate on this one because CI failed to compile.  I'll put WIP there.

Reply:
bazel format  fixed this.